wasp/src/main/java/com/orhanobut/wasp/WaspResponse.java

```diff
@@ -11,12 +11,12 @@
  */
 final class WaspResponse {
 
-    private final String url;
```

Comment:
I really don't like the idea of making fields exposed, even though most of them are immutable. We should access them by getter methods. with package-private modifiers